tests/openQA_containers/build.pm

```diff
@@ -0,0 +1,13 @@
+use strict;
+use base "openQAcoretest";
```

Comment:
you miss `use warnings` here! or is it intentional? i dont see it in other modules as well!!
you can also use `use Mojo::Base qw(openQAcoretest);` and remove the `strict` and `warnings` as the Mojo::Base import them for you.

Reply:
Done